aircraft/performance_tutorial.py

```diff
@@ -0,0 +1,104 @@
+""" A simple aircraft performance model to be used as GPkit tutorial"""
+from gpkit import Variable, Model
+from gpkit.tools import te_exp_minus1
+import numpy as np
+
+# Free Variables
+b      = Variable('b', 'm', 'Wing span')
+CD     = Variable('C_D', '-', 'Drag coefficient')
+CL     = Variable('C_L', '-', 'Lift coefficient')
+LDl    = Variable('\\frac{L}{D}_land', '-', 'Lift-to-drag in final descent')
+M      = Variable('M', '-', 'Mach number')
+Mdd    = Variable('M_{dd}', '-', 'Drag divergence Mach number')
+R      = Variable('R', 'nautical_miles', 'Range')
+S      = Variable('S', 'm^2', 'Wing reference area')
+s_GR   = Variable('s_{GR}', 'm', 'Ground roll distance')
+s_TO   = Variable('s_{TO}', 'm', 'Total takeoff distance')
+sl     = Variable('sl', 'm', 'Total landing distance')
+sl_gr  = Variable('sl_{gr}', 'm', 'Ground roll distance (landing)')
+sl_oc  = Variable('sl_{oc}', 'm' , 'Obstacle clearance distance (landing)')
+T      = Variable('T', 'N', 'Thrust')
+T_TO   = Variable('T_{TO}', 'N', 'Takeoff thrust')
+V      = Variable('V', 'knots', 'Velocity')
+Vst    = Variable('V_{stall}', 'm/s', 'Stall speed')
+W_fuel = Variable('W_{fuel}', 'lbf', 'Fuel weight')
+W      = Variable('W', 'lbf', 'Initial gross weight')
+z_bre  = Variable('z_{bre}', '-', 'Breguet parameter')
+
+# Fixed Parameters
+a     = Variable('a', 300, 'm/s', 'Speed of sound at 33,000 ft')
+AR    = Variable('AR', 11 , '-', 'Aspect Ratio')
+bmax  = Variable('b_{max}', 40, 'm', 'Gate constraint')
+cosL  = Variable('\\cos(\\Lambda)', np.cos(25*np.pi/180), '-',
+                 'Cosine of quarter chord sweep')
+CD0   = Variable('CD0', 0.02, '-', 'Parasitic drag coefficient')
+CDl   = Variable('C_D_{land}', 0.08, '-', 'Drag coefficient in landing configuration')
+CLmaxt= Variable('C_{L_{max}}', 2.0, '-', 'Maximum lift coefficient (takeoff)')
+CLmaxl= Variable('C_{L_{max}}', 3.0, '-', 'Maximum lift coefficient (landing)')
```

Comment:
no need to change this for tutorial tomorrow, but feels like the takeoff and landing conditions might be natural application for VectorVariables.